src/wizard/flutter/index.md

```diff
@@ -94,6 +94,8 @@ To learn more about the API and automatic instrumentations, check out the [perfo
 
 We offer a range of methods to provide Sentry with debug symbols so that you can see symbolicated stack traces and triage issues faster.
 
+Check out [Debug Symbols](/platforms/flutter/upload-debug/) documentation to upload debug symbols for Android, iOS/macOS and source maps for Web to Sentry.
```

Comment:
That is repeated now since I added:

> You'll also need to [upload the debug information files](/platforms/flutter/upload-debug/) generated by the `flutter` build for iOS, macOS, and Android NDK native crashes.

Just after, lets just remove it?